lab_7_llm/main.py

```diff
@@ -94,6 +128,7 @@ def __getitem__(self, index: int) -> tuple[str, ...]:
         Returns:
             tuple[str, ...]: The item to be received
         """
+        return "|".join((self._data.iloc[index]["premise"], self._data.iloc[index]["hypothesis"])),
```

Comment:
Why join? You need to take a data using a `iloc`, then take the required columns, convert the values ​​to strings and return a tuple of them

Reply:
done